### PR TITLE
Add initial skeleton of the statement of rights

### PR DESCRIPTION
**Pull request by Firstyear**
Implements #78, documentation on rights and ethics for consumers of our software.

- [ - ] cargo fmt has been run
- [ - ] cargo test has been run and passes
- [ x ] design document included (if relevant)

### CODE REVIEW

**Comment by Firstyear** [comment]
@excitedleigh @ryankurte It would be great to have you review and potential refactor or expand this. It's a bit of a rough draft, but I think it's the kind of idea I want to go for in this document.

File: ethics/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Rights of people
+
+All people using this software should expect to have the right to:
+
+* Ability to be forgotten
+* Self determination and autonomy
+* Free from harmful discrimination of any kind
+* Informed consent and Privacy of their data
+* To use and access this software regardless of ability
```
Comment (ryankurte) [inline ethics/README.md]: how about something like "to access and understand data held and shared on their behalf", intended to support access to their account audit trail with logins / information access / whatever else is created through their interaction with the service, in a supporting GDPR or NZ Privacy Act kinda way?

File: ethics/EXAMPLES.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+People should be able to change their own name at anytime. Consider divorce, leaving abusive partners
+or other personal decisions around why a name change is relevant.
+
+This is why names are self-service writeable at any time.
+
+## Free from harmful discrimination of any kind
+
+### Cultural and Social awareness of name formats
+
+All name fields should be case sensitive utf8 with no max or min length limit. This is
```
Comment (ryankurte) [inline ethics/EXAMPLES.md]: naive question, do you _need_ more than one name field (like, is there a benefit to the first name / lastname / whatever arrangement vs. just having one name field)? just that some people that will want it?

and, on the basis that all these fields are customisable anyway (afaik from talking to you about this, might be incorrect), is there a way that you could recommend but also gently influence this (with a default configuration or something)?

Comment (Firstyear) [reply ethics/EXAMPLES.md]: Ahhh name fields are displayName/legalName. They have different privacy settings by default in the project. But I should make taht clearer (there is plenty of documentation about why firstname/lastname is not usable).

Comment (JJJollyjim) [reply ethics/EXAMPLES.md]: One of my biggest frustrations with software is the name/preferredName structure, which guides people into always using the wrong one. I definitely like your approach.

**Pull request merged**